### PR TITLE
Merge snapshots to reduce code size

### PR DESCRIPTION
**Pull request by galpeter**
As of fe26674 JerryScript can merge multiple snapshots
into one and execute them by index. By merging snapshots
the code size (rodata) is reduced by ~10 kb on arm/x86_64.

JerryScript-DCO-1.0-Signed-off-by: Peter Gal pgal.u-szeged@partner.samsung.com

### CODE REVIEW

File: src/iotjs_binding.c [hunk 1]
```diff
@@ -446,17 +447,16 @@ iotjs_jval_t iotjs_jhelper_eval(const char* name, size_t name_len,
...
    * until jerry_cleanup is not called */
-
-  *throws = jerry_value_has_error_flag(res);
-
-  jerry_value_clear_error_flag(&res);
-
-  return iotjs_jval_create_raw(res);
+  *throws = jerry_value_has_error_flag(result);
+  jerry_value_clear_error_flag(&result);
+  return iotjs_jval_create_raw(result);
```
Comment (daeyeon) [inline src/iotjs_binding.c]: I guess that `iotjs_jval_create_raw` is planned to be removed. Can we simply do `return result`?

Comment (LaszloLango) [reply src/iotjs_binding.c]: Yes, please fix it.

**Comment by galpeter** [comment]
@daeyeon, @LaszloLango Okay, will remove the `_raw` call.

**Comment by galpeter** [comment]
@daeyeon, @LaszloLango Updated

**Comment by yichoi** [comment]
LGTM

**Pull request merged**